troposphere/settings/default.py

```diff
@@ -98,8 +98,10 @@
 # code execution vulnerability. So keep this.
 SESSION_SERIALIZER = 'django.contrib.sessions.serializers.JSONSerializer'
 
+SESSION_COOKIE_NAME = 'tropo_sessionid'
+
 # default cookie age to # of seconds in 2 days
-#SESSION_COOKIE_AGE = (2 * 24 * 60 * 60)
+SESSION_COOKIE_AGE = (2 * 24 * 60 * 60)
```

Comment:
This is disabling during a hasty triage. Re-enabling seems to make the most sense as default is 2 weeks.